spec/ruby/core/string/chilled_string_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../../spec_helper'
+
+describe "chilled String" do
+  guard -> { ruby_version_is "3.4" and !"test".equal?("test") } do
+    describe "#frozen?" do
+      it "returns true" do
+        "chilled".frozen?.should == true
+      end
+    end
+
+    describe "#-@" do
+      it "returns a different instance" do
+        input = "chilled"
+        interned = (-input)
+        interned.frozen?.should == true
+        interned.object_id.should_not == input.object_id
+      end
+    end
+
+    describe "#+@" do
+      it "returns a different instance" do
+        input = "chilled"
+        duped = (+input)
+        duped.frozen?.should == false
+        duped.object_id.should_not == input.object_id
+      end
+    end
+
+    describe "mutation" do
+      it "emits a warning" do
+        input = "chilled"
+        -> {
+          input << "mutated"
+        }.should complain(/literal string will be frozen in the future/)
+      end
+
+      it "raises FrozenError after the string was explictly frozen" do
+        input = "chilled"
+        input.freeze
+        -> {
+          input << "mutated"
+        }.should raise_error(FrozenError)
+      end
+    end
+
+    it "is not ractor shareable" do
+      input = "chilled"
+      Ractor.shareable?(input).should == false
+    end
+
+    it "can be made ractor shareable" do
+      input = "chilled"
+      shareable = Ractor.make_shareable(input)
+      shareable.should == input
+      Ractor.shareable?(shareable).should == true
+    end
```

Comment:
Could you move the Ractor-related tests to test-all?
We avoid testing Ractor in ruby/spec for reasons explained [here](https://github.com/ruby/spec/pull/1043#issuecomment-1930313473), in short Ractor is at least currently a CRuby-specific API.

Reply:
Thanks. Done.